src/server/models/okta/User.ts

```diff
@@ -117,7 +117,6 @@ export enum Status {
 	PROVISIONED = 'PROVISIONED',
 	RECOVERY = 'RECOVERY',
 	PASSWORD_EXPIRED = 'PASSWORD_EXPIRED',
-	PASSWORD_RESET = 'PASSWORD_RESET',
```

Comment:
If this enum doesn't match the return results of the Okta API could we potentially get a type error somewhere if a user is in a `PASSWORD_RESET` state?

Reply:
thankfully the enum isn't used directly by zod, so it shouldn't lead to a type error somewhere

we have error handling which handles cases where the user is in an unknown user state, e.g. 

https://github.com/guardian/gateway/blob/4b58d6ee26ae1c01ba9d66001899167bd49cc4bf/src/server/controllers/sendChangePasswordEmail.ts#L347-L350